service/pom.xml

```diff
@@ -115,8 +116,36 @@
 					<groupId>edu.berkeley.compbio</groupId>
 					<artifactId>jlibsvm</artifactId>
 				</exclusion>
+				<exclusion>
+					<groupId>org.aksw.semweb2nl</groupId>
+					<artifactId>triple2nl</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>javax.servlet</groupId>
+					<artifactId>servlet-api</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.apache.httpcomponents/httpmime -->
+		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpmime</artifactId>
+		</dependency>
+
+
+		<dependency>
+			<groupId>org.dice-research</groupId>
+			<artifactId>triple2nl</artifactId>
+			<version>0.4.1-SNAPSHOT</version>
```

Comment:
Do we even need Bengal as a dependency if you use triple2nl directly?

Reply:
I think I left bengal dependency to preserve the previous way that COPAAL was verbalizing in `DefaultPathVerbalizer`.